### PR TITLE
Bump boostrap compiler to new beta

### PR DESCRIPTION
**Pull request by BoxyUwU**
Accidentally left some comments on the update cfgs commit directly xd

### CODE REVIEW

**Review by albertlarsan68 — APPROVED** [review]
LGTM

**Comment by albertlarsan68** [comment]
Thanks for the PR!
@bors r+ rollup=never

**Comment by cuviper** [comment]
:ghost: **boo**strap

**Comment by Kobzol** [comment]
Let's prioritize this a bit.

@bors priority=1

File: src/stage0
Comment (onur-ozkan) [inline src/stage0:1]: Depending on the merge order, this or https://github.com/rust-lang/rust/pull/130045 will get overridden.

Comment (Kobzol) [reply src/stage0:1]: This PR is against `master`, https://github.com/rust-lang/rust/pull/130045 is against `beta`.

Comment (onur-ozkan) [reply src/stage0:1]: Missed that, thanks.

**Pull request merged**